### PR TITLE
Link to deployment of PKI page

### PR DESCRIPTION
**Pull request by joinimran**
As suggested by user that content is missing in the document, I have linked the page with the deployment of PKI certificate. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6360

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The minimum required Enterprise certificate authority that can be used with Wind
 * The certificate template must have an extension that has the value "DomainController", encoded as a [BMPstring](https://docs.microsoft.com/windows/win32/seccertenroll/about-bmpstring). If you are using Windows Server Enterprise Certificate Authority, this extension is already included in the domain controller certificate template.
 * The domain controller certificate must be installed in the local computer's certificate store.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md:77]:
```suggestion
* The domain controller certificate must be installed in the local computer's certificate store. See [Step-by-step example deployment of the PKI certificates](https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/network/example-deployment-of-pki-certificates) for details.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ The minimum required Enterprise certificate authority that can be used with Wind
...
 
- 
+See [Step-by-step example deployment of the PKI certificates](https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/network/example-deployment-of-pki-certificates).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-trust-prereqs.md:79]:
```suggestion
```

**Comment by joinimran** [comment]
@JohanFreelancer9  - Thanks for the correction. Sorry I missed removing of locale from the link. Thanks for reminding me.

Imran.

**Comment by joinimran** [comment]
@e0i - PR is ready for merge.

**Comment by e0i** [comment]
Related to #6358

**Comment by mapalko** [comment]
The link that is added to the doc is for SCCM. I am not sure that is relevant here. I did not see the submitter referencing that doc in his comments. I know there are additional instructions for DC certificate set up in the cert trust section: https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-pki

**Comment by joinimran** [comment]
@mapalko - The linked doc covers many aspects of certificates deployment. The problem submitter doesn't reply for the exact link so I put my suggestions but those to see if that helps.

**Comment by mapalko** [comment]
Since the current linked page isn't specific to WHFB, I'd prefer to link the WHFB guidance.

**Comment by joinimran** [comment]
@mapalko - I have changed the link. See if that is ok.

**Review by illfated — APPROVED** [review]
Looks just fine me. BTW, this PR would benefit from more attention, when everyone is back from their holidays.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**